doc/whats-new.rst

```diff
@@ -33,6 +33,12 @@ v0.11.0 (unreleased)
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+- The option ``enable_cftimeindex`` has now been set to ``True`` by default.
```

Comment:
We should think about eventually removing this option entirely, and making it simply a "no-op" for now (raising FutureWarning). In the long term I think we would prefer to avoid keeping around extraneous global options.

Reply:
Makes sense -- that will clean up the datetime decoding logic (and tests) quite a bit.